roles/auth/tasks/main.yml

```diff
@@ -46,6 +51,15 @@
     mozilla/fxa-auth-server node scripts/gen_vapid_keys.js
     creates=/data/config/auth/vapid-keys.json
 
+- name: generate oauth/auth openid key file, and dummy oldkeyfile
+  become: true
+  command: >-
+    docker run --rm --net=host -v /data/config/openid:/openid
+      -e FXA_OPENID_KEYFILE=/openid/openid-key.json
+      -e FXA_OPENID_OLDKEYFILE=/openid/openid-old-key.json
+    mozilla/fxa-auth-server node fxa-oauth-server/scripts/gen_keys.js
```

Comment:
I just deleted fxa-oauth-server from master today. the new location for this script is `scripts/oauth_gen_keys.js`